src/App.js

```diff
@@ -84,6 +85,47 @@ const PageNavigation = (params) => {
   }
 };
 
+const AdCluster = (params) => {
```

Comment:
As the number of components increases descriptive names for components and variables becomes even more important. Naming is very difficult, First lets describe what this component does and what role it plays. I see this as a container/view/page of ad cluster summaries. Would you agree? Does it have other roles or behavior? How would you describe this component?

Reply:
I think it is a container for displaying a collection of ad cluster summaries. So maybe AdClustersContainer or AdClustersDisplay would be better ?

Reply:
Yeah, those sound great to me. I'm leaning toward AdClustersDisplay.